### PR TITLE
deps: cleanup & upgrade

### PR DESCRIPTION
**Pull request by Kompwu**
This is a cherry-pick of #1164 w/ resolved conflicts & upgrade.
Closes #1164

### CODE REVIEW

**Comment by Kompwu** [comment]
Ping @nchanged

**Comment by Kompwu** [comment]
@nchanged It's been a week...

File: package.json [hunk 1]
```diff
@@ -33,93 +33,76 @@
...
+    "prettysize": "1.1.0",
     "realm-utils": "^1.0.9",
-    "regexpu-core": "^4.1.3",
-    "request": "^2.79.0",
+    "request": "^2.87.0",
     "shorthash": "0.0.2",
-    "source-map": "^0.7.1",
-    "tslib": "^1.8.0",
-    "watch": "^1.0.1",
-    "ws": "^1.1.1"
```
Comment (nchanged) [inline package.json]: That's a huge bump from 1 to 5
Did you verify the build yourself?

Comment (Kompwu) [reply package.json]: Which one?

**Comment by nchanged** [comment]
closing this for now. Conflicts + breaking the builds

**Pull request closed without merging**